simtools/visualization/legend_handlers.py

```diff
@@ -308,6 +323,78 @@ def legend_artist(self, legend, orig_handle, fontsize, handlebox):
         return patch
 
 
+class HESSHandler(TelescopeHandler):
```

Comment:
Again, all of the following functions are exactly the same? There must be a better way (sorry to be annoying).

Reply:
See reply above